### DISCUSSION TITLE
Can I subscribe to a project using Microsoft Teams?

### DISCUSSION THREAD

**Discussion by balcamo** [community/community#37906] [Projects and Issues]
I'm trying to subscribe a team on Microsoft Teams to a Project so we get notifications when draft issues are created but I can't seem to get it to work. Any suggestions?

**Comment by ossanna16** [comment]
Hi there @balcamo and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by nearlythere** [comment]
Did the documentation help?
https://github.com/integrations/microsoft-teams#subscribe-notifications

Which Teams integration are you using? 

Please note there's two apps in the wild. 

"GitHub for Teams by Microsoft Corporation"
https://appsource.microsoft.com/en-us/product/office/WA200002077?exp=ubp8&tab=Overview
- Last updated 10/4/2022
- Fully supported by GitHub, it's the one they recommend using. 
- Has official documentation https://github.com/integrations/microsoft-teams#subscribe-notifications 

Or this one called "GitHub Enterprise by Microsoft Teams Ecosystem"
https://appsource.microsoft.com/en-us/product/office/WA104381552?exp=ubp8&tab=Overview
- Has fewer features
- Was last updated in 1/21/2020
- The privacy policy link is broken. 
- Smells like abandonware(?)
